README.md

```diff
@@ -101,6 +101,59 @@ add `priority` label when you start backing us :smile:
 
 See: [docs/deployment.md](docs/deployment.md)
 
+# Development
+
+> This short guide is meant to give a guide to how github apps are made, and basic steps to reproduce the OpenCollective Bot should you wush to fix any bug.
+
+... Note Probot was used to build the Github app hence certain details may be specific to **Probot**
+
+## Installation
+
+Feel free to clone or download this project using the **git clone** and install using **Yarn install** or **Npm install** to install required dependencies
+
+Github apps make a large use of **WebHooks**. Hence, you can use [Ngrok](https://ngrok.com) , [Smee](https://smee.io) or other comfortable options .
+
+### Running Ngrok
+
+> After making all necessary local Ngrok installations found here [Ngrok](https://ngrok.com)
+
+- Run `Ngrok http <{localhost port}>` and copy the web address it gives into the your `.env` file
```

Comment:
What should be the name of the ENV variable in the environment file?

Reply:
I think the web address key ngrok generates on startup

Reply:
This doesn't answer my question. You give the instruction to "copy the web address" to the `.env` file. What should be the name of the environment variable in there?